confluence/models/space.py

```diff
@@ -41,8 +41,8 @@ def __init__(self, json):  # type: (Dict[str, Any]) -> None
             pass  # TODO - Description comes back with `view` & `plain` expandable, not clear whether that's a common object so not handling for now
 
         # Homepage is an expandable full page object
-        if 'homepage' in json:
-            self.homepage = Page(json['homepage'])
+        #if 'homepage' in json:
+        #    self.homepage = Page(json['homepage'])
```

Comment:
Rather than commenting it out, if this is causing issues then can you late import Page here inside the if statement?